scripts/generate-ligo.sh

```diff
@@ -48,10 +49,10 @@ tez_wrapper_sources=(
   error
   common
   fixedPoint
-  kit
-  lqt
   fa2Interface
   fa2Ledger
+  kit
+  lqt
```

Comment:
Now that I look at it actually, I can't find a reason to have `kit`, `lqt`, or even `fixedPoint` in this list at the moment (i.e., I see no dependency on those for the tez wrapper) :thinking: Not necessarily important for this PR but we should probably remove them.